index.html

```diff
@@ -146,72 +183,120 @@ <h3 class="card-h">Profesional Art <br> Printing Data</h3>
-            <ul id="title1">
-                <li><img src="assets/square2.png" alt="diamond-shapped bullet point">&nbsp; Language &nbsp;&nbsp;&nbsp; </li>
-                <li class="items">Javascript</li>
-                <li class="items">Ruby</li>
-                <li class="items">Html</li>
-                <li class="items">CSS</li>
-            </ul>
-    
-            <ul id="title2">
-                <li><img src="assets/square1.png" alt="squared-shapped bullet point">&nbsp; Frameworks &nbsp;&nbsp;&nbsp; </li>
-                <li class="items">Bootstrap</li>
-                <li class="items">Ruby on Rails</li>
-                <li class="items">RSPec</li>
-                <li class="items">CapyBara</li>
-                <li class="items">Selenium</li>
-            </ul>
-        
-            <ul id="title3">
-                <li><img src="assets/round.png" alt="round-shapped bullet point">&nbsp; Skills &nbsp;&nbsp;&nbsp; </li>
-                <li class="items">Codekit</li>
-                <li class="items">Github</li>
-                <li class="items">Codepen</li>
-                <li class="items">Gitlab</li>
-                <li class="items">Terminal</li>
-            </ul>
-    </div> 
+        <ul id="title1">
+            <li><img src="assets/square2.png" alt="diamond-shapped bullet point">&nbsp; Language &nbsp;&nbsp;&nbsp;
+            </li>
+            <li class="items">Javascript</li>
+            <li class="items">Ruby</li>
+            <li class="items">Html</li>
+            <li class="items">CSS</li>
+        </ul>
+
+        <ul id="title2">
+            <li><img src="assets/square1.png" alt="squared-shapped bullet point">&nbsp; Frameworks &nbsp;&nbsp;&nbsp;
+            </li>
+            <li class="items">Bootstrap</li>
+            <li class="items">Ruby on Rails</li>
+            <li class="items">RSPec</li>
+            <li class="items">CapyBara</li>
+            <li class="items">Selenium</li>
+        </ul>
+
+        <ul id="title3">
+            <li><img src="assets/round.png" alt="round-shapped bullet point">&nbsp; Skills &nbsp;&nbsp;&nbsp; </li>
+            <li class="items">Codekit</li>
+            <li class="items">Github</li>
+            <li class="items">Codepen</li>
+            <li class="items">Gitlab</li>
+            <li class="items">Terminal</li>
+        </ul>
+    </div>
+    <section class="contact-me">
+        <h3>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h3>
+        <div class="form">
+            <form action="https://formspree.io/f/mrgjjnpd" method="post"></form>
+            <input id="name" type="text" placeholder="Full Name" maxlength="30" required>
+            <input id="email" type="email" name="email" placeholder="Email address">
+            <textarea id="message" name="message" placeholder="Enter text here" maxlength="500"></textarea>
+            <input id="submit" type="submit" value="Get in touch">
```

Comment:
- [ ] Well done on the design 👍 , however, your form is not working now and the reason is that you forgot to wrap all your inputs inside od the form, please put the closing tag of your form after the submit input and your form will work as expected